webpack.config.js

```diff
@@ -14,7 +13,7 @@ const common = {
     output: {
         path: path.resolve(__dirname, "dist"),
 		//dev: specify a full path including protocol, production: specify full path excluding protocol
-        publicPath: config.publicPath,
+        publicPath: ".",
```

Comment:
Please revert this change. You can avoid errors by duplicating the file `config.example.js` and renaming it to `config.js`. Don't forget to set your public path in `config.js` to `.` then.